charts/rancher-cis-benchmark-crd/6.1.0-rc1/templates/clusterscanprofile.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: clusterscanprofiles.cis.cattle.io
+spec:
+  group: cis.cattle.io
+  names:
+    kind: ClusterScanProfile
+    plural: clusterscanprofiles
+  scope: Cluster
```

This file with https://github.com/rancher/cis-operator/blob/master/crds/clusterscanprofile.yaml:

```suggestion
    plural: clusterscanprofiles
    singular: clusterscanprofile
  scope: Cluster
```

---

done